bin/tessel-2.js

```diff
@@ -70,6 +71,28 @@ parser.command('install-drivers')
   })
   .help('Install drivers');
 
+parser.command('crash-reporting-off')
+  .callback(function() {
+    crashReporter.turnOff()
+      .then(module.exports.closeSuccessfulCommand, module.exports.closeFailedCommand);
+  })
+  .help('Turn off crash reporting completely');
+
+parser.command('crash-reporting-on')
+  .callback(function() {
+    crashReporter.turnOn()
+      .then(module.exports.closeSuccessfulCommand, module.exports.closeFailedCommand);
+  })
+  .help('Turn on crash reporting');
+
+parser.command('crash-reporting-check')
```

Comment:
Yes, it was only meant for tests. I am not sure where integration tests go. If you have any suggestions, please let me know.